pkg/configuration/parser.go

```diff
@@ -304,21 +304,18 @@ func (p *nsxConfigParser) getEndpointsFromScopePaths(groupsPaths []string) ([]to
 		// in scope - "any" are all the vms
 		return p.configRes.VMs, p.allGroups // all endpoints and groups
 	}
-	endPoints, groups, _ := p.getEndpointsFromGroupsPaths(groupsPaths, false)
-	return endPoints, groups
+	ruleEndPoints := p.getEndpointsFromGroupsPaths(groupsPaths, false)
```

Comment:
```suggestion
	ruleEndpoints := p.getEndpointsFromGroupsPaths(groupsPaths, false)
```
rename `EndPoints` to `Endpoints` across the relevant places